docfiles/docs.js

```diff
@@ -279,6 +280,110 @@ function renderSnippets() {
     });
 }
 
+function languageOption(code) {
+    var locale = pxt.Util.allLanguages[code];
+    if (!locale)
```

Comment:
Lets remove these check. We should crash instead of silently succeed if the apptheme locale is not a subset of all available locales